config/locales/views/en.yml

```diff
@@ -585,6 +585,15 @@ en:
       contact_support: contact Support
       last_renamed: You last changed your user name on %{renamed_at}.
       more_info: For information on how changing your user name will affect your account, please check out the %{account_faq_link}. Note that changes to your user name may take several days or longer to appear. If you are still seeing your old user name on your works, bookmarks, series, or collections after a week, please %{contact_support_link}.
+    confirmation:
+      contact_support: contact Support
+      go_back: Return to Archive front page
+      must_activate_html:
+        one: "<strong>Important!</strong> You must activate your account within %{count} day, or your account will expire. If this happens, you can sign up again using the same invitation."
```

Comment:
The [wiki](https://github.com/otwcode/otwarchive/wiki/Internationalization-(i18n)-Standards) says:
> Keep HTML in the view files. This will reduce the risk of encountering issues with an email's markup when the locale files are automatically updated.

Does the wiki need changing to clarify when HTML tags are allowed to go in the locale files, or does this need some revision?